cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt

```diff
@@ -2,5 +2,4 @@ psutil==5.9.4
 rfc3339>=6.2
 grpcio==1.41.1
 googleapis-common-protos==1.6.0
-tensorflow==2.9.3; platform_machine=="x86_64"
-tensorflow-aarch64==2.9.1; platform_machine=="aarch64"
+tensorflow==2.11.0
```

Comment:
Since Tensorflow v2.10.0, the binary for aarch64 is shipped as `tensorflow`.

> aarch64 CPUs: Experimental performance optimizations from [Compute Library for the Arm® Architecture (ACL)](https://github.com/ARM-software/ComputeLibrary) are available through oneDNN in the default Linux aarch64 package (pip install tensorflow).

https://github.com/tensorflow/tensorflow/releases/tag/v2.10.0